osidb_bindings/session.py

```diff
@@ -297,26 +330,30 @@ def update(self, id, form_data: Dict[str, Any]):
             sync_fn = get_sync_function(method_module)
             return sync_fn(
                 id,
+                *args,
                 client=self.client(),
                 form_data=transformed_data,
                 multipart_data=UNSET,
                 json_body=UNSET,
+                **kwargs,
             )
         else:
             raise OperationUnsupported(
                 'Operation "update" is not supported for the '
                 f'"{self.resource_name}" resource.'
             )
 
-    def delete(self, id):
+    def delete(self, id, *args, **kwargs):
```

Comment:
a (very) random and unrelated observation - are cascade deletes in data service layer also removing comments/references ?

Reply:
Yes, cascade deletes are set.